catalog/views.py

```diff
@@ -1,3 +1,37 @@
 from django.shortcuts import render
 
 # Create your views here.
+from .models import Book, Author, BookInstance, Genre
+
+def index(request):
+    """View function for home page of site."""
+
+    # Generate counts of some of the main objects
+    num_books = Book.objects.all().count()
```

Comment:
đếm số bản ghi thì `.count()` luôn em
sửa tương tự ở dưới
```suggestion
    num_books = Book.objects.count()
```

Reply:
dạ i18n là em liệt kê các text bên view xong bên html em add nó vô {{}} ạ chị, v là sai r ạ